python/cuml/test/test_pca.py

```diff
@@ -76,15 +82,19 @@ def test_pca_fit_transform(datatype, input_type):
         Xcupca = cupca.fit_transform(X)
 
     assert array_equal(Xcupca, Xskpca, 1e-3, with_sign=True)
+    if stream is not None:
```

Comment:
Don't we need to sync in any case? Also is the sync not needed before line 84?

Reply:
Done